src/App.css

```diff
@@ -13,42 +13,42 @@ orange: f4a431
 dark-orange: c65e26
 green: 366e41
 dark-green: 173e2e
+whitesmoke/light gray: 	f5f5f5
 */
 
 body {
   background-color: #173e2e;
   background-image: url('img/background.jpg');
   height: 100vh;
+  width: 100vw;
   background-size: cover;
   background-position: center;
   text-align: center;
   display: grid;
   align-items: center;
-  justify-content: center;
-  color: whitesmoke;
+  justify-content: stretch;
+  color: black;
 }
 
-.soon {
-  color: red;
-}
-
-.kind-of-soon {
-  color: orange;
+.main {
+  width: 30vw;
+  background-color: rgba(245, 245, 245, 0.85);
+  padding: 1rem;
+  font-family: 'Source Sans Pro', sans-serif;
+  border-radius: 1rem;
+  width: 90%;
+  margin: 2% auto;
 }
 
-.not-so-soon {
-  color: green;
+@media only screen and (min-width: 736px) {
+  .main {
+    max-width: 600px;
+  }
 }
 
-.inactive {
-  color: gray;
-}
-.modal {
-  position: fixed;
-  width: 100%;
-  height: 100%;
-  background-color: rgba(0, 0, 0, 0.25);
-}
+/* 
+NEEDS TO BE REMOVED TO ANOTHER CSS FILE.
```

Comment:
There's nothing like today to get this done 🙂

Reply:
Just a head's up . . .
This gets handled in @lacrivella 's modal css branch.